js/output/sql/sql-output.js

```diff
@@ -350,7 +352,9 @@ window.SQLOutput = Backbone.View.extend({
 
         var output = Handlebars.templates["sql-results"]({
             tables: tables,
-            results: results
+            results: results,
+            databaseMsg: i18n._("Database Schema"),
+            resultsMsg: i18n._("Query results"),
```

Comment:
I changed from "Results" to "Query results" for more uniqueness in Crowdin

Reply:
Oh, this is great, thanks for thinking about this. <3